pkg/actors/actors.go

```diff
@@ -344,7 +371,11 @@ func (a *actorsRuntime) Call(ctx context.Context, req *invokev1.InvokeMethodRequ
 	var err error
 
 	if a.isActorLocal(targetActorAddress, a.config.HostAddress, a.config.Port) {
-		resp, err = a.callLocalActor(ctx, req)
+		if a.isInternalActor(actor.GetActorType()) {
```

Comment:
What if it is not a local actor? Will internal actors also work as remote?

Reply:
So, does that mean that the internal actors can also be invoked remotely?

Reply:
Correct. Internal actors can be invoked locally or remotely.

Reply:
Maybe @johnewart  can pick this up and implement this feature in dapr/dapr.